frontend/src/components/ErrorState/ErrorState.tsx

```diff
@@ -119,7 +122,7 @@ export const ErrorState = ({
 						</Space>
 					)}
 				</div>
-			</ElevatedCard>
+			</Callout>
```

Comment:
Is this the the main UI change in the PR?

Reply:
Yeah. That was it! Just switching them out was enough.